File: Java/Carrots.java
import java.util.Scanner;

public class Carrots {
	public static void main(String args[]) {
		int numContestants = 0, numProblems = 0;
		Scanner input = new Scanner(System.in);
		numContestants = input.nextInt();
		numProblems = input.nextInt();
		if (numContestants < 1 || numProblems > 1000) System.exit(1);
		
		String contestantDescriptions[] = new String[numContestants];
		for (int i = 0; i < numContestants; i++) {
			Scanner desc = new Scanner(System.in);
			contestantDescriptions[i] = desc.next();
		}
		
		System.out.printf(numProblems + "\n");
	}
}

File: Python/Carrots.py
#!/usr/env/python
def main():
  #inputList = input()
  numContestants,numProblems=map(int,raw_input().split(' '))
  
  if (numContestants < 1) or (numProblems > 1000):
    exit(1)
  
  contestantDescriptions = [numContestants]
  for i in range(0,numContestants - 1):
    contestantDescriptions[i] = raw_input()
  
  print numProblems
  exit(0)

if __name__ == "__main__":
  main()